### PR TITLE
Add Contributing guidelines and PR template to the repo

### PR DESCRIPTION
**Pull request by SamDepardieu**
⬆️ Says it all. 

We need these file to open source the repository in a short future. They are the same as for the reference parser 👍

### CODE REVIEW

**Comment by SamDepardieu** [comment]
[Create a PR template for the scraper](https://trello.com/c/9jCnTF9j/368-create-a-pr-template-for-the-scraper)

**Comment by SamDepardieu** [comment]
[Create a contribution template for the web scraper](https://trello.com/c/9fMd0fw6/366-create-a-contribution-template-for-the-web-scraper)

File: pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+# Checklist:
+
+- [ ] My code follows the style guidelines of this project (pep8 AND pyflakes)
+- [ ] I have commented my code, particularly in hard-to-understand areas
+- [ ] If needed, I changed related parts of the documentation
+- [ ] I included tests in my PR
+- [ ] New and existing unit tests pass locally with my changes
+- [ ] Any dependent changes have been merged and published in downstream modules
+- [ ] If my PR aims to fix an issue, I referenced it using `#(issue)`
```
Comment (hblanks) [inline pull_request_template.md]: All good, though for a fledgling project, sometimes we might keep the process even simpler for better adoption. No need to change here, though.

**Comment by SamDepardieu** [comment]
For the License, we need to check with Danil what he wants for this project. I'd go MIT, but I think Danil didn't like the idea of non-open sourse reuse of the project, so we may have to go for Gnu GPL3

**Pull request merged**